package.json

```diff
@@ -11,24 +11,36 @@
     "node": ">=12.16.1"
   },
   "scripts": {
+    "dev": "lerna run start --scope @blitzjs/* --stream --parallel",
     "build": "lerna exec --stream -- yarn build",
     "lerna": "lerna",
     "lerna-prepare": "lerna run prepare",
     "start": "lerna exec --parallel -- yarn start",
     "test": "lerna exec --parallel -- yarn test"
   },
   "devDependencies": {
+    "@types/debug": "^4.1.5",
+    "@types/jest": "^25.1.3",
+    "@types/node": "^13.7.4",
     "cross-env": "^7.0.0",
     "husky": "^4.2.3",
     "jest": "^25.1.0",
     "lerna": "^3.20.2",
     "lint-staged": "^10.0.7",
     "npm-run-all": "^4.1.5",
     "prettier": "^1.19.1",
+    "pretty-quick": "2.0.1",
     "rimraf": "^3.0.2",
-    "typescript": "^3.7.5",
     "tsdx": "^0.12.3",
     "tslib": "^1.10.0",
-    "@types/jest": "^25.1.3"
+    "typescript": "^3.7.5"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged"
+    }
+  },
+  "dependencies": {
+    "debug": "^4.1.1"
```

Comment:
This likely should not be declared as a dependency in the root `package.json`. Typically, only common `devDependencies` should be pulled up to the root.